### PR TITLE
Drag n drop

### PR DESCRIPTION
**Pull request by FBerthelot**
## JIRA

https://jira.jahia.org/browse/PROJECT-XXX

## Tests

The following are included in this PR
- [ ] Unit Tests (Most changes _should_ have unit tests)
- [ ] Integration Tests

## Checklist

I have considered the following implications of my change: 

- [ ] Security (in particular for changes to authentication, authorization, data fetching, ...)
- [ ] Performance
- [ ] Migration
- [ ] Code maintainability

## Documentation

- [ ] Inline documentation
- [ ] Internal Documentation (wiki)
- [ ] User-facing Documentation

### CODE REVIEW

File: src/javascript/EditPanel/EditPanelContent/FormBuilder/ChildrenSection/ChildrenContainer/DragDrop/DropableSpace.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.dropable {
+  padding: 8px 0;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+
```
Comment (Taybou) [inline src/javascript/EditPanel/EditPanelContent/FormBuilder/ChildrenSection/ChildrenContainer/DragDrop/DropableSpace.scss]: remove space :3

File: src/javascript/EditPanel/EditPanelContent/FormBuilder/ChildrenSection/ChildrenContainer/DragDrop/DropableSpace.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        const item = {name: 'tata'};
+        useDrop.mockImplementation(obj => {
+            obj.drop(item);
+            return [{}];
+        });
+
+        const handleReorder = jest.fn();
+
+        shallowWithTheme(
+            <DropableSpace index={42} childUp={null} childDown={null} onReorder={handleReorder}/>,
```
Comment (Taybou) [inline src/javascript/EditPanel/EditPanelContent/FormBuilder/ChildrenSection/ChildrenContainer/DragDrop/DropableSpace.spec.jsx]: why not 21? :D

File: src/javascript/EditPanel/EditPanelContent/FormBuilder/ChildrenSection/ChildrenContainer/ChildrenContainer.spec.jsx [hunk 3]
```diff
@@ -40,28 +40,25 @@ describe('ChildrenContainer component', () => {
         }
     };
 
-    it('should display an article', () => {
+    it('should display an the title', () => {
```
Comment (Taybou) [inline src/javascript/EditPanel/EditPanelContent/FormBuilder/ChildrenSection/ChildrenContainer/ChildrenContainer.spec.jsx]: weird text!

File: src/javascript/EditPanel/EditPanelContent/FormBuilder/ChildrenSection/ChildrenContainer/DragDrop/DropableSpace.spec.jsx (hunk 2 above)
Comment (FBerthelot) [reply src/javascript/EditPanel/EditPanelContent/FormBuilder/ChildrenSection/ChildrenContainer/DragDrop/DropableSpace.spec.jsx]: https://fr.wikipedia.org/wiki/2001,_l%27Odyss%C3%A9e_de_l%27espace

File: src/javascript/EditPanel/EditPanelContent/FormBuilder/ChildrenSection/ChildrenContainer/DragDrop/DropableSpace.scss (hunk 1 above)
Comment (FBerthelot) [reply src/javascript/EditPanel/EditPanelContent/FormBuilder/ChildrenSection/ChildrenContainer/DragDrop/DropableSpace.scss]: Nice catch, I will add the sass linter :)

**Pull request merged**